drivers/volume/portworx/portworx.go

```diff
@@ -2421,7 +2421,7 @@ func (p *portworx) StartMigration(migration *storkapi.Migration) ([]*storkapi.Mi
 			return nil, fmt.Errorf("error getting list of volumes to migrate: %v", err)
 		}
 		for _, pvc := range pvcList.Items {
-			if !p.OwnsPVC(core.Instance(), &pvc) {
+			if !p.IsSupportedPVC(core.Instance(), &pvc, true) {
```

Comment:
nit: is this part of a different bug / PR ? How is this related to resource transformation?

Reply:
yes included this included as separate commit  https://github.com/libopenstorage/stork/pull/1130/commits/75b4a674f7de72e498c4ea2a0d6b671939046fb4